tonic_validate/utils/telemetry.py

```diff
@@ -22,10 +22,26 @@
 
 class Telemetry:
     def __init__(self, api_key: Optional[str] = None):
+        """
+        The Telemetry class is used to log telemetry data to the Tonic Validate server.
+
+        Parameters
+        ----------
+        api_key: Optional[str]
+            The API key to use for authentication
+        """
         self.config = Config()
         self.http_client = HttpClient(self.config.TONIC_VALIDATE_TELEMETRY_URL, api_key)
 
     def get_user(self) -> UserInfo:
+        """
+        Gets the user info from the file or creates a new user if it does not exist.
```

Comment:
Retrieves the user information from the file. If the user does not exist, creates a new user.